platform/android/docs/lat-lng-bounds.md

```diff
@@ -0,0 +1,8 @@
+# Using the LatLngBounds camera API
+
+!!! info
+    This guide is a work in progress.
```

Comment:
Do we already have a backlog item to flesh this out?

Reply:
I'll include it in this PR.